index.css

```diff
@@ -190,22 +208,51 @@ body {
   margin-top: 60px;
 }
 
+#current-img {
+  width: 160px;
+  height: 100px;
+  mix-blend-mode: multiply;
+}
+
+#first-day_img,
+#second-day_img,
+#last-day_img {
+  width: 65px;
+  height: 40px;
+}
+
+.sunny-img {
+  object-fit: contain;
+  object-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
 .content-info_weather__temperature {
   font-size: 70px;
   color: white;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 700;
+  font-style: normal;
 }
 
 .content-info_weather__characteristics {
   margin-left: 25px;
   color: white;
   font-size: 22px;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 700;
+  font-style: normal;
 }
```

Comment:
names definetely against BEM notation.
https://ru.bem.info/methodology/quick-start/#%D0%BF%D1%80%D0%B8%D0%BD%D1%86%D0%B8%D0%BF%D1%8B-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%8B-%D1%81-%D1%8D%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82%D0%B0%D0%BC%D0%B8

and further:
<img width="892" alt="image" src="https://github.com/user-attachments/assets/d13ca896-0fea-49af-8ab8-340b3cdb841c">

Also check AZDS internal styleguide
https://azdsgroup.atlassian.net/wiki/x/YQBruw